metrics/bigquery.py

```diff
@@ -170,16 +174,18 @@ def update_query(self, config):
                 last_time = int(time.time() - (60*60*24*self.backfill_days))
         else:
             # InfluxDB is not enabled so skip backfill, just query for last day.
-            last_time = int(time.time() - (60*60*24))
+            last_time = int(time.time() - (60*60*24)*self.backfill_days)
```

Comment:
If we make this change and influxdb is not enabled the job will recalculate results for the entire backfill period every day. Some of these queries are rather large so that could be expensive.

Reply:
SGTM